package/MDAnalysis/analysis/rdf.py

```diff
@@ -411,16 +537,15 @@ def _single_frame(self):
                                                     box=self.u.dimensions)
 
             for j, (idx1, idx2) in enumerate(pairs):
-                self.count[i][idx1, idx2, :] += np.histogram(dist[j],
-                                                             **self.rdf_settings)[0]
+                self.results.count[i][idx1, idx2, :] += np.histogram(dist[j],
+                    **self.rdf_settings)[0]
```

Comment:
The original indentation was clearer. If PEP8 complains about line too long, ignore PEP8 – readability is better than always following PEP8.